crates/astria-conductor/src/block_verifier.rs

```diff
@@ -414,12 +478,136 @@ fn get_proposer(validator_set: &ValidatorSet) -> eyre::Result<Validator> {
 
 #[cfg(test)]
 mod test {
-    use std::str::FromStr;
+    use std::{
+        collections::BTreeMap,
+        str::FromStr,
+    };
 
-    use tendermint::block::Commit;
+    use astria_sequencer_validation::{
+        generate_action_tree_leaves,
+        MerkleTree,
+    };
+    use base64::engine::{
+        general_purpose::STANDARD,
+        Engine as _,
+    };
+    use tendermint::{
+        block::Commit,
+        validator,
+    };
 
     use super::*;
 
+    fn make_test_validator_set(height: u32) -> ValidatorSet {
+        use ed25519_consensus::SigningKey;
+
+        let key_bytes = STANDARD.decode("XXhEZctO2gj3JhjHV7214N53zEl3s05eD9cWIJJHAVrPP2L5ARCQ0PSkNA85f3avKwpBsaDD6Q6VJlhN6ZHKjQ==").unwrap();
```

Comment:
Are the magic bytes here important? Could we get away with just generating the key randomly?